### PR TITLE
Events updates for issue 439

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: draft/2022-04-20-this-week-in-rust.md [hunk 1]
```diff
@@ -148,28 +148,18 @@ decision. Express your opinions now.
...
     * [**Remote Book Club: Rust for Rustaceans Chapter Discussion**](https://www.meetup.com/RustPhilly/events/284644487)
 * 2022-04-20 | San Diego, CA, US | [San Diego Rust](https://www.meetup.com/San-Diego-Rust/)
     * [**San Diego Rust April 2022 Tele-Meetup**](https://www.meetup.com/San-Diego-Rust/events/284705301/)
 * 2022-04-20 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
     * [**Rust Movie/Study/Hack/Hang-out night**](https://www.meetup.com/Vancouver-Rust/events/nwcmpsydcgbbc/)
+* 2022-04-20 | Cardiff, UK | [Rust and C++ Cardiff](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/)
+    * [**Rust image viewer from scratch**](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/events/285193324/)
 * 2022-04-21 | Charlottesville, VA, US | [Charlottesville Rust Meetup](https://www.meetup.com/Charlottesville-Rust-Meetup/)
     * [**The Rusty Boss--directing the world with Rust, Bluetooth, and a Raspberry Pi**](https://www.meetup.com/Charlottesville-Rust-Meetup/events/284922970)
 * 2022-04-23 | Various - EMEA | [Rustfest](https://rustfest.world/)
```
Comment (mariannegoldin) [inline draft/2022-04-20-this-week-in-rust.md:165]:
```suggestion
```
(RustFest LATAM Cancelled per this [announcement](https://rustfest.world/news/emea-edition-cancellation)

Comment (mariannegoldin) [inline draft/2022-04-20-this-week-in-rust.md:165]:
```suggestion
```
(RustFest LATAM Cancelled per this [announcement](https://rustfest.world/news/emea-edition-cancellation)

Comment (mariannegoldin) [reply draft/2022-04-20-this-week-in-rust.md:165]: (Note, not sure why Github won't let me suggest a removal of both lines for this announcement, but that is the desired change)

Comment (andrewpollack) [reply draft/2022-04-20-this-week-in-rust.md:165]: Good catch -- removing both lines now

File: draft/2022-04-20-this-week-in-rust.md [hunk 2]
```diff
@@ -198,26 +192,16 @@ Rusty Events between 2022-04-13 - 2022-05-11 🦀
     * [**Monthly meetup**](https://www.meetup.com/Seattle-Rust-Meetup/events/gskksrydchbnb/)
 * 2022-05-11 | Boulder, CO, US | [Boulder Elixir and Rust](https://www.meetup.com/boulder-elixir-rust/)
     * [**Monthly Meetup**](https://www.meetup.com/boulder-elixir-rust/events/zvxcsrydchbpb/)
+* 2022-05-18 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
```
Comment (mariannegoldin) [inline draft/2022-04-20-this-week-in-rust.md:195]:
```suggestion
* 2022-05-12 | Stuttgart, DE | [Rust Community Stuttgart](https://www.meetup.com/Rust-Community-Stuttgart/)
    * [**Rust-Meetup**](https://www.meetup.com/Rust-Community-Stuttgart/events/swgrssydchbqb/)
* 2022-05-18 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
```

File: draft/2022-04-20-this-week-in-rust.md [hunk 3]
```diff
@@ -198,26 +192,16 @@ Rusty Events between 2022-04-13 - 2022-05-11 🦀
...
 
-* 2022-04-13 | Atlanta, GA, US | [Rust ATL](https://www.meetup.com/Rust-ATL/)
-    * [**Grab a beer with fellow Rustaceans**](https://www.meetup.com/Rust-ATL/events/pczdssydcgbrb/)
-* 2022-04-14 | Columbus, OH, US | [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
-    * [**Monthly Meeting**](https://www.meetup.com/columbus-rs/events/dpkhgrydcgbsb/)
-* 2022-04-19 | San Francisco, CA, US | [San Francisco Rust Study Group](https://www.meetup.com/san-francisco-rust-study-group/)
-    * [**Rust Hacking in Person**](https://www.meetup.com/san-francisco-rust-study-group/events/wjkjssydcgbzb/)
 * 2022-04-27 | Boston, MA, US | [Boston Rust Meetup](https://www.meetup.com/BostonRust/)
     * [**Boston Rust Meetup at Amazon**](https://www.meetup.com/BostonRust/events/284808948)
 * 2022-05-11 | Atlanta, GA, US | [Rust ATL](https://www.meetup.com/Rust-ATL/)
```
Comment (mariannegoldin) [inline draft/2022-04-20-this-week-in-rust.md:207]:
```suggestion
* 2022-05-11 | Atlanta, GA, US | [Rust ATL](https://www.meetup.com/Rust-ATL/)
* 2022-05-12 | Columbus, OH, US | [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
    * [**Monthly Meeting**](https://www.meetup.com/columbus-rs/events/dpkhgrydchbqb/)
* 2022-05-17 | San Francisco, CA, US | [San Francisco Rust Study Group](https://www.meetup.com/san-francisco-rust-study-group/)
    * [**Rust Hacking in Person**](https://www.meetup.com/san-francisco-rust-study-group/events/wjkjssydchbwb/)
```

**Comment by mariannegoldin** [comment]
This looks good! I made a couple suggestions to add some more events.

File: draft/2022-04-20-this-week-in-rust.md [hunk 4]
```diff
@@ -148,44 +148,38 @@ decision. Express your opinions now.
...
-    * [**Mob Programming: Rome Tools**](https://www.meetup.com/RustPhilly/events/bgqgtsydcgbxb/)
-* 2022-04-19 | Washington, DC, US | [Rust DC](https://www.meetup.com/RustDC/)
-    * [**Mid-month Rustful**](https://www.meetup.com/RustDC/events/vdhxgsydcgbzb/)
 * 2022-04-20 | Philadelphia, PA, US | [Rust Philly (Rust Philadelphia)](https://www.meetup.com/RustPhilly/)
     * [**Remote Book Club: Rust for Rustaceans Chapter Discussion**](https://www.meetup.com/RustPhilly/events/284644487)
 * 2022-04-20 | San Diego, CA, US | [San Diego Rust](https://www.meetup.com/San-Diego-Rust/)
     * [**San Diego Rust April 2022 Tele-Meetup**](https://www.meetup.com/San-Diego-Rust/events/284705301/)
 * 2022-04-20 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
     * [**Rust Movie/Study/Hack/Hang-out night**](https://www.meetup.com/Vancouver-Rust/events/nwcmpsydcgbbc/)
+* 2022-04-20 | Cardiff, UK | [Rust and C++ Cardiff](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/)
```
Comment (mariannegoldin) [inline draft/2022-04-20-this-week-in-rust.md:161]: Would recommend placing the Cardiff event first in the April 20 dates, due to the alphabetization scheme we are thinking about for the events section (Date, then City name ascending alphabetically)

**Comment by mariannegoldin** [comment]
Look great! Approving

**Pull request merged**